### ISSUE TITLE
crypto module missing des-ede3 cipher

### ISSUE THREAD

**Issue by xland** [electron/electron#27277]
### Preflight Checklist
* [x] I have read the [Contributing Guidelines](https://github.com/electron/electron/blob/master/CONTRIBUTING.md) for this project.
* [x] I agree to follow the [Code of Conduct](https://github.com/electron/electron/blob/master/CODE_OF_CONDUCT.md) that this project adheres to.
* [x] I have searched the issue tracker for an issue that matches the one I want to file, without success.

### Issue Details

* **Electron Version:**
  * 11.1.1  &  10.1.3
* **Operating System:**
  * Windows 10 

### Expected Behavior
support `des-ede3` cipher

### Actual Behavior
```js
var encrypto = require("crypto")
encrypto.getCiphers()
```

> 0: "aes-128-cbc"
1: "aes-128-cfb"
2: "aes-128-ctr"
3: "aes-128-ecb"
4: "aes-128-gcm"
5: "aes-128-ofb"
6: "aes-192-cbc"
7: "aes-192-ctr"
8: "aes-192-ecb"
9: "aes-192-gcm"
10: "aes-192-ofb"
11: "aes-256-cbc"
12: "aes-256-cfb"
13: "aes-256-ctr"
14: "aes-256-ecb"
15: "aes-256-gcm"
16: "aes-256-ofb"
17: "des-cbc"
18: "des-ecb"
19: "des-ede"
20: "des-ede-cbc"
21: "des-ede3-cbc"
22: "rc2-cbc"
23: "rc4"

**Comment by xland** [comment]
And Electron 11.2.1

**Comment by nornagon** [comment]
des-ede3 is an alias for des-ede3-cbc, which BoringSSL already supports.

**Comment by xland** [comment]
@nornagon 

Is that true?

I try these code  and got  different output:
```js
let crypto = require("crypto")
let des = (param) => {
  let cipher = crypto.createCipheriv(param.alg, param.key, param.iv);
  cipher.setAutoPadding(true)
  let ciph = cipher.update(param.plaintext, 'utf8', 'base64');
  ciph += cipher.final('base64');
  return ciph
}
let a = des({
  alg: 'des-ede3',
  key: Buffer.from('123456789012345678901234'),
  iv: Buffer.alloc(0),
  plaintext: 'roomid=123456',
})
let b = des({
  alg: 'des-ede3-cbc',
  key: Buffer.from('123456789012345678901234'),
  iv: Buffer.alloc(8),  //must be 8 or Uncaught Error: Invalid IV length
  plaintext: 'roomid=123456',
})
console.log(a) // V+rZCcfYJY6ecvRzljI8+w==
console.log(b) // V+rZCcfYJY570BDbXVt57g==
```

**Comment by nornagon** [comment]
Ah, sorry, you're right. I misread. From openssl:

```
310: des-ede3-cbc       Three key triple DES EDE in CBC mode
311: des-ede3           Three key triple DES EDE in ECB mode
```

It looks like boringssl [supports des-ede3](https://boringssl.googlesource.com/boringssl/+/refs/heads/master/crypto/fipsmodule/cipher/e_des.c#223), so we should be able to expose it.

**Comment by nornagon** [comment]
@xland I made no such claim :) It was landed in the 12-x-y branch with 189f7e0e4e588624994b57f4fc96b93d443eae47, which has not been included in any release yet. See https://github.com/electron/electron/compare/v12.0.0...12-x-y
